### DISCUSSION TITLE
Retorno de erros "response: [IncomingMessage]"

### DISCUSSION THREAD

**Discussion by olavomello** [banzeh/cielo#122] [General]
Olá Flávio,

antes de mais nada obrigado. Lib fantástica e muito prática.
Estou tentando tratar erros porém recebo o mesmo problema. 
"response: [IncomingMessage]" 
Sem informações que permitam o debug mais aprofundado, o que dificulta bastante a identificação e correção.

Seria possível ampliar os dados deste erro ?

Obrigado

**Comment by banzeh** [comment]
@olavomello  beleza?

Cara, fiz uns testes aqui e consegui obter a resposta da Cielo usando o seguinte código:

```js
.catch((err) => {
    console.error(err.response.Code, err.response.Message);
});
```

No meu teste retornou a seguinte informação:

`126 Credit Card Expiration Date is invalid`

Com o response.Code você consegue obter o código de erro padronizado pela Cielo.

PS: Fiz o teste no ambiente de sandbox. Atualmente não tenho acesso ao ambiente de prod da Cielo.
